engine/src/main/java/org/terasology/engine/modes/loadProcesses/InitialiseWorld.java

```diff
@@ -133,8 +133,8 @@ public boolean step() {
         context.get(ComponentSystemManager.class).register(new RelevanceSystem(chunkProvider), "engine:relevanceSystem");
         EntityAwareWorldProvider entityWorldProvider = new EntityAwareWorldProvider(
                 new WorldProviderCoreImpl(worldInfo, chunkProvider, blockManager.getBlock(BlockManager.UNLOADED_ID),
-                        context)
-                , context);
+                        context),
```

Comment:
Again, I'd rather extract local variables here. The `WorldProviderCoreImpl`, the unloaded block instance, etc.